addons/hr/models/res_users.py

```diff
@@ -361,3 +362,44 @@ def action_related_contact(self):
             'res_model': 'res.partner',
             'view_mode': 'form',
         }
+
+    def _to_store_avatar_card_employee_prefetch_fields(self):
+        if not self.env.user._is_internal():
+            return []
+        base_fields = [
+            "work_email",
+            "work_phone",
+            "work_location_id",
+            "department_id",
+            "company_id",
+        ]
+        if self.env.user.has_group("hr.group_hr_user"):
+            # job_title is not a field of hr.employee.public, but it is a field of hr.employee
+            base_fields.extend(
+                [
+                    "job_title",
+                ]
+            )
```

Comment:
in general: make sure to have single lines as much as possible (when it fits formatter line length)